bundles/org.eclipse.chronograph.api/src/org/eclipse/chronograph/api/Group.java

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************************
+ * Copyright (c) 2020 ArSysOp
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * https://www.eclipse.org/legal/epl-2.0/.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Sergei Kovalchuk <sergei.kovalchuk@arsysop.ru> - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.chronograph.api;
+
+/**
+ * 
+ * The Group describes typed container with sequence Brick items  
+ *
+ */
+public interface Group extends BrickContainer {
```

Comment:
Any reference to its Container?

Reply:
The Group describes typed container with sequence Brick items